Realm/RLMSyncManager.h

```diff
@@ -79,14 +79,6 @@ typedef void(^RLMSyncErrorReportingBlock)(NSError *, RLMSyncSession * _Nullable)
 @property (nonatomic, copy) NSString *appID;
 
 /**
- Whether SSL certificate validation should be disabled. SSL certificate validation is ON by default. Setting this
```

Comment:
Okay, sure. If you don't set it we'll use the value from the configuration. If you do set it we'll log a warning to console and then use the global value instead.